docs/asus-aura-led-guide.md

```diff
@@ -0,0 +1,92 @@
+
+
+## Initialization
+
+Asus Aura LED controller does not need to be initialized before use. Initialization is optional and recommended.
+
+```
+# liquidctl -m Aura initialize
+AsusTek Aura LED Controller
+└── Firmware version    AULA3-AR32-0207
+```
+
+## Status
+
+The `status` function currently returns a list of 6-byte values whose meanings are not fully understood. This information is provided nevertheless in the hopes of encouraging someone to interpret them properly.
+```
+% liquidctl -m Aura status
+AsusTek Aura LED Controller
+├── Device Config: 1     0x1e, 0x9f, 0x03, 0x01, 0x00, 0x00
+├── Device Config: 2     0x78, 0x3c, 0x00, 0x01, 0x00, 0x00
+├── Device Config: 3     0x78, 0x3c, 0x00, 0x01, 0x00, 0x00
+├── Device Config: 4     0x78, 0x3c, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 5     0x00, 0x00, 0x00, 0x01, 0x04, 0x02
+├── Device Config: 6     0x01, 0xf4, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 7     0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 8     0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+├── Device Config: 9     0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+└── Device Config: 10    0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+```
+
+## RGB lighting
+
+There is one 12V RGB channel named `rgb` and three 5V Addressable RGB channels named `argb1`, `argb2`, and `argb3`. Because the driver uses `effect` mode, all channels are synchronized. It is not possible to set different color modes to different channels except through `direct` mode.
+
+```
+# liquidctl -m Aura set rgb color static af5a2f
+# liquidctl -m Aura set argb1 color breathing 350017
+# liquidctl -m Aura set argb2 color rainbow
+# liquidctl -m Aura set argb3 color spectrum-cycle
+# liquidctl -m Aura set sync color gentle-transition
+```
+
+Colors can be specified in RGB, HSV or HSL (see [Supported color specification formats](../README.md#supported-color-specification-formats)), and each animation mode supports zero or one color. 
+
+
+| Mode | Colors | Notes |
+| --- | --- | --- |
+| `off` | None |
+| `static` | One |
+| `breathing` | One |
+| `flashing` | One |
+| `spectrum_cycle` | None |
+| `rainbow` | None | 
+| `spectrum_cycle_breathing` | None |
+| `chase_fade` | One |
+| `spectrum_cycle_chase_fade` | None |
+| `chase` | One |
+| `spectrum_cycle_chase` | None |
+| `spectrum_cycle_wave` | None |
+| `chase_rainbow_pulse` | None |
+| `rainbow_flicker` | None |
+| `gentle_transition` | None | name given by us |
+| `wave_propagation` | None | name given by us |
+| `wave_propagation_pause` | None | name given by us |
+| `red_pulse` | None | name given by us |
+
+In addition to these, it is also possible to use the `sync` pseudo-channel to apply a setting to all lighting channels.
+
+
+## Correspondence between lighting channels and physical locations
+
+Each user may need to create a table that associates generic channel names to specific areas or headers on their motherboard. 
```

Comment:
Will the channel names not always map to the same header?

Reply:
Correct. This section is not necessary and has been removed.